run/interpolaters.py

```diff
@@ -0,0 +1,75 @@
+import math
+import numpy as np
+from scipy import interpolate
+import pyproj
+
+# Returns TOPAZ data interpolated from the data grid and coordinates to the target grid and coordinates
+def topaz4_interpolate(target_lon, target_lat, data, data_x, data_y, proj_string):
```

Comment:
Suggest making the preceding comment into a docstring. Similarly for the other functions
```suggestion
def topaz4_interpolate(target_lon, target_lat, data, data_x, data_y, proj_string):
    """
    Returns TOPAZ data interpolated from the data grid and coordinates to the target grid and coordinates
    """
```